docs/getting-started.rst

```diff
@@ -22,6 +22,15 @@ To validate the installation, start a Python shell or a Jupyter notebook and typ
 
 .. code-block:: python
 
-    import gettsim
-
-    gettsim.test()
+    import pandas as pd
+    from gettsim.config import ROOT_DIR
+    from gettsim.tax_transfer import tax_transfer
+    from gettsim.policy_for_date import get_policies_for_date
+
+    tax_data = yaml.safe_load((ROOT_DIR / "data" / "param.yaml").read_text())
```

Comment:
YAML should be hidden from the user. Please include that in the `get_policies_for_date` function.

Reply:
Included loading the standard tax data as default in `get_policies_for_date`